modules/service/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -894,18 +894,19 @@ input GmosNorthGratingConfigInput {
 }
 
 # Edit or create GMOS North Long Slit advanced configuration
-input GmosNorthLongSlitAdvancedConfigInput {
-  # The overrideWavelength field may be unset by assigning a null value, or ignored by skipping it altogether
-  overrideWavelength: WavelengthInput
+input GmosNorthLongSlitInput {
```

Comment:
Collapses advanced and basic into one input for GMOS North long slit and one for GMOS South long slit.  The input allows you to

* change the grating, filter and/or FPU
* override the central wavelength that would otherwise be used by default
* override the exposure time mode from the science requirements
* optionally set explicit values for x-binning, etc. that are otherwise defaulted

Reply:
Could we just update the exposure time mode in the science requirements directly? And not have an override? Might be something for Andy to answer.

Reply:
That sounds good to me.  I'm going to assume we can do that for now.  I think it implies that in the requirements section we'll have to allow them the option to pick fixed exposure time/count.

Reply:
Right. In best case we remove it from the old advanced configuration tab and only have it in the requirements.